src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -48,8 +48,15 @@ struct PathManagerParams
         internal IPathResolver PathResolver { get; set; }
     }
 
-    class PathManager : IPathManager
+    public sealed class PathManager : IPathManager
```

Comment:
@mjkkirschner Is this API breaking? But we really need this change...

Reply:
I'm not sure, I think the default if not specified in most cases is actually `internal` - but why the heck do you need to make it public and sealed?

Reply:
I think `seal` can be skipped, but if let's say an integrator like DynamoPlayer wants to just use the instance to access Preferences which is one of their ask right now, this class needs to be public